build.gradle.kts

```diff
@@ -72,6 +93,31 @@ allprojects {
 
     // Define the repositories universally for all modules, including the root.
     repositories.standardToSpineSdk()
+
+    val spine = io.spine.internal.dependency.Spine
+    val validation = io.spine.internal.dependency.Validation
+    val protoData = io.spine.internal.dependency.ProtoData
+    val logging = io.spine.internal.dependency.Spine.Logging
+
+    doForceVersions(configurations)
+    configurations {
```

Comment:
Please see if it's possible to move this whole new section to `doForceVersions)`.